firebolt_provider/hooks/firebolt.py

```diff
@@ -50,15 +50,20 @@ def get_ui_field_behaviour() -> Dict:
         """Returns custom field behaviour"""
         return {
             "hidden_fields": ["port"],
-            "relabeling": {"schema": "Database", "host": "API End Point"},
+            "relabeling": {
+                "schema": "Database",
+                "login": "Username",
+                "extra": "Advanced Connection Properties",
+            },
             "placeholders": {
-                "host": "firebolt api end point",
-                "schema": "firebolt database",
-                "login": "firebolt userid",
-                "password": "password",
+                "host": "The host name of your Firebolt database",
+                "schema": "The database to connect to",
+                "login": "The email address you use to log into Firebolt",
+                "password": "Your password to log in to Firebolt",
                 "extra": json.dumps(
                     {
-                        "engine_name": "firebolt engine name",
+                        "account_name": "firebolt",
```

Comment:
Should this contain a hint on what account name is, like other fields?

Reply:
yep, changed it